common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -433,39 +444,11 @@ def parse_query_params(strategy, response, *args, **kwargs):
     if not (auth_entry and auth_entry in _AUTH_ENTRY_CHOICES):
         raise AuthEntryError(strategy.backend, 'auth_entry missing or invalid')
 
-    # Note: We expect only one member of this dictionary to be `True` at any
-    # given time. If something changes this convention in the future, please look
-    # at the `login_analytics` function in this file as well to ensure logging
-    # is still done properly
```

Comment:
Quick note about the history of this comment.  The current text makes it sound like this assumption applies only to the analytics step in the pipeline, but the original comment (before we added the analytics step) was just "We expect only one member of this dictionary to be `True` at any given time." -- meaning that we make this assumption throughout the pipeline.

I'd definitely recommend manual testing this on the registration/login page.  The unit tests are okay, but in practice the pipeline can behave in unexpected ways.  We have some test plans from when we originally launched this feature that could be helpful.